library/core/src/intrinsics.rs

```diff
@@ -719,6 +719,7 @@ extern "rust-intrinsic" {
     ///
     /// A more user-friendly and stable version of this operation is
     /// [`std::process::abort`](../../std/process/fn.abort.html).
+    #[rustc_const_unstable(feature = "const_abort", issue = "none")]
```

Comment:
Please explain that this is for the const panic machinery, and not actually implemented (I assume actually reaching this during CTFE will ICE)